### ISSUE TITLE
where does gt.zip come from, and  any suggest of this result?

### ISSUE THREAD

**Issue by bygreencn** [whai362/PSENet#26]
1. Train with ICDAR2015 Ch4 with 600 epoch and batch_size change to 32, the log result looks like this:
0.000010	0.394331	0.889751	0.852450	
0.000010	0.416218	0.898702	0.863312	
0.000010	0.406228	0.874345	0.837213	
0.000010	0.378316	0.900977	0.864393
2.Testing found the result some intrersting labels like this:
![image](https://user-images.githubusercontent.com/22853/55533528-1f597780-56e4-11e9-988d-543ec76be4db.png)
3. Eval result is 
Calculated!{"recall": 0.0, "precision": 0.0, "hmean": 0, "AP": 0}
Where is wrong, all is following with README, only change the batch_size from 16 to 32, and the train result fit your log recorder.

**Comment by float123** [comment]
Excuse me, who solved this problem? I used the pre-training model downloaded from Baiduyun to test it. This is also the result.
![img_1](https://user-images.githubusercontent.com/39334643/56559152-53350800-65d3-11e9-8b08-31c95f250d58.jpg)

**Comment by float123** [comment]
> Excuse me, who solved this problem? I used the pre-training model downloaded from Baiduyun to test it. This is also the result.
> ![img_1](https://user-images.githubusercontent.com/39334643/56559152-53350800-65d3-11e9-8b08-31c95f250d58.jpg)

找到原因了，是路径没设置对

**Comment by Mary63** [comment]
> 1. Train with ICDAR2015 Ch4 with 600 epoch and batch_size change to 32, the log result looks like this:
>    0.000010	0.394331	0.889751	0.852450
>    0.000010	0.416218	0.898702	0.863312
>    0.000010	0.406228	0.874345	0.837213
>    0.000010	0.378316	0.900977	0.864393
> 
> 2.Testing found the result some intrersting labels like this:
> ![image](https://user-images.githubusercontent.com/22853/55533528-1f597780-56e4-11e9-988d-543ec76be4db.png)
> 
> 1. Eval result is
>    Calculated!{"recall": 0.0, "precision": 0.0, "hmean": 0, "AP": 0}
>    Where is wrong, all is following with README, only change the batch_size from 16 to 32, and the train result fit your log recorder.



Hi :)

How did you draw these green rectangles? After training and testing, I get the .txt files, but then I don't know how to see these rectangles on my pictures. 

Any help would be greatly appreciated.

Mary
